cmd/iam-store.go

```diff
@@ -1661,7 +1661,8 @@ func (store *IAMStoreSys) SetTempUser(ctx context.Context, accessKey string, cre
 		_, combinedPolicyStmt := filterPolicies(cache, mp.Policies, "")
 
 		if combinedPolicyStmt.IsEmpty() {
-			return time.Time{}, fmt.Errorf("specified policy %s, not found %w", policyName, errNoSuchPolicy)
```

Comment:
%w allows for error wrapping @jiuker that is why `errors.Is(err, errNoSuchPolicy)` works - this PR is not correct. 

what are you trying to fix here?